changelogs/unreleased/gh-8927-socketpair.md

```diff
@@ -0,0 +1,4 @@
+## feature/lua/socket
```

Comment:
Typo in the commit message/PR description:

```lua
local socket = require('socket')
local s1, s1 = socketpair('AF_UNIX', 'SOCK_STREAM', 0)
```

`socketpair` -> `socket.socketpair`.